runtime/makelib/copyright.c.in

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+ 
+/* Autogenerated code. A copyright string must be included in each DLL and executable */
+
+const char j9copyright[] =   "Copyright (c) 1991, @J9VM_CURRENT_YEAR@ IBM Corp. and others\n\n"
```

Comment:
Why the different format for the `@J9VM_CURRENT_YEAR@` tag here vs other places?  ie: https://github.com/eclipse/openj9/blob/b07109b08416afbe346045878809155aba8323a8/runtime/include/j9cfg.h.in#L37

Reply:
Freemarker vs cmake syntax